include/verilated.cpp

```diff
@@ -278,9 +278,7 @@ Verilated::Serialized::Serialized() {
     s_timeprecision = VL_TIME_PRECISION;  // Initial value until overriden by _Vconfigure
 }
 
-void Verilated::NonSerialized::setup() {
-    s_profThreadsFilenamep = strdup("profile_threads.dat");
-}
+void Verilated::NonSerialized::setup() { s_profThreadsFilenamep = strdup("profile_threads.dat"); }
```

Comment:
Should I just let clang-format do whatever it wants?  I didn't touch some of these lines, but clang-format did.

Reply:
If your version doesn't match then we might start fighting each other, so I'd suggest no, don't merge anything which you didn't touch please.

Reply:
I got clang-format-10 and still see these changes (even when running just against master).  I'm running:
```
make clang-format CLANGFORMAT=clang-format-10
```